packages/angular/projects/ui-angular/src/lib/components/amplify-sign-in/amplify-sign-in.component.ts

```diff
@@ -25,12 +26,18 @@ export class AmplifySignInComponent
   implements AfterContentInit, OnInit, OnDestroy
 {
   @HostBinding('attr.data-amplify-authenticator-signin') dataAttr = '';
-  @Input() public headerText = 'Sign in to your account';
+  @Input() public headerText = I18n.get(DefaultTexts.SIGN_IN_TEXT);
```

Comment:
Oops, that's a typo! I meant to do `translate('Sign in to your account')`. 

But it can be done either way -- both are type checked. I personally prefer hard coded string because it's more visible what the string is, but `DefaultTexts.SIGN_IN_TEXT` works just as well.

Reply:
👍